http_service/tests/test_schedule_tests.py

```diff
@@ -74,6 +79,20 @@ def test_simple_schedule(patch_resources, mock_hgmo, mock_repo):
             "NOK",
             ["Base history 2", "Base history 1", "Base history 0"],
         ),
+        # patch that only applies after a pull has been done
```

Comment:
Can you add another case with `autoland` instead of `try`

Reply:
Done